File: backend/Properties.Domain/Entities/Property.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Properties.Domain.Entities
{
    public class Property
    {
        public Guid IdProperty { get; set; } = default!;
        public string Name { get; set; } = default!;
        public string Address { get; set; } = default!;
        public decimal Price { get; set; } = default!;
        public string CodeInternal { get; set; } = default!;
        public int Year { get; set; }
        public string OwnerId { get; set; } = default!;
    }
}

File: frontend/src/types/property.ts
export type Property = {
    idProperty?: string;
    name: string;
    address: string;
    price: number;
    codeInternal: string;
    year: number;
    idOwner: string;
    image: string;
};

export type PropertyDetail = {
    idProperty?: string;
    ownerId: string;
    name: string;
    address: string;
    price: number;
    codeInternal: string;
    year: number;
    image: string;
    trace: Trace[];
};

export type Trace = {
    idTrace: string;
    dateSale: string;
    name: string;
    value: number;
    tax: number;
}

export type CreatePropertyRequest = {
    name: string;
    address: string;
    price: number;
    year: number;
    image: string;
    idOwner: string;
};

export type GetPropertyRequest = {
    idProperty: string;
};